stdlib/LinearAlgebra/docs/src/index.md

```diff
@@ -527,67 +527,105 @@ the input argument belongs on (`side`). The possibilities are:
 | `'N'`       | The diagonal values of the matrix `X` will be read.       |
 | `'U'`       | The diagonal of the matrix `X` is assumed to be all ones. |
 
+
 ```@docs
 LinearAlgebra.BLAS
+LinearAlgebra.BLAS.set_num_threads
+LinearAlgebra.BLAS.get_num_threads
+```
+
+### Level 1 BLAS functions
```

Comment:
If you are adding these subheadings I think you should explain what it means.

Reply:
OK, I've added some simple explanations and links.